### PR TITLE
Upgrade redis and minor fixes for scaling up containers

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR
 - upgrade django-redis
 - adds PURGE_DATA boolean param to OMERO-server job to preserve DB
 - upgrade selenium containers to version 3 (work in progress, this is failing anyway)

Also some minor adjustments to the containers.

### CODE REVIEW

File: web/Dockerfile [hunk 1]
```diff
@@ -57,5 +57,11 @@ RUN chmod a+x /tmp/run.sh
 ARG USER_ID=1000
 RUN usermod -u $USER_ID omero
 
+# make sure mounted volumes has correct permissions
+RUN mkdir -p /home/omero/ngin
```
Comment (manics) [inline web/Dockerfile]: `ngin` -> `nginx`?

Comment (aleksandra-tarkowska) [inline web/Dockerfile]: sorry I haven;t tested volumes yet, I will work on that

**Comment by joshmoore** [comment]
This needs to be sync with recent devspace changes from @sbesson

**Comment by aleksandra-tarkowska** [comment]
if this is merged please tag devspace

**Pull request merged**